core/src/main/java/uk/ac/bbsrc/tgac/miso/core/manager/UserAuthMisoRequestManager.java

```diff
@@ -2317,4 +2337,29 @@ public void deleteBox(Box box) throws IOException {
       throw new IOException("User " + getCurrentUser().getFullName() + " cannot change Box " + box.getAlias());
     }
   }
+
+  @Override
+  public long saveSequencerServiceRecord(SequencerServiceRecord record) throws IOException {
+    if (getCurrentUser().isAdmin()) {
+      return backingManager.saveSequencerServiceRecord(record);
+    }
+    else {
+      throw new IOException("User " + getCurrentUser().getFullName() + " cannot write to this Service Record");
+    }
+  }
+
+  @Override
+  public SequencerServiceRecord getSequencerServiceRecordById(long id) throws IOException {
+    return backingManager.getSequencerServiceRecordById(id);
+  }
+
```

Comment:
readCheck/writeCheck are for objects types that have more specific permissions to them than "all users" or "admin only." Get/list methods for other readable-by-all object types in this class follow the same pattern as this ( https://github.com/oicr-gsi/miso-lims/blob/GLT-212_sequencer_ref_details/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/manager/UserAuthMisoRequestManager.java#L2180 )